pkg/api/v1/defaults.go

```diff
@@ -372,3 +372,17 @@ func SetDefaults_ScaleIOVolumeSource(obj *v1.ScaleIOVolumeSource) {
 		obj.FSType = "xfs"
 	}
 }
+
+func SetDefaults_QuobyteVolumeSource(obj *v1.QuobyteVolumeSource) {
+	if obj.User == "" {
+		obj.User = "root"
```

Comment:
these defaults are new, and change the already-released API... what was the behavior of the user and group fields if no values were set previously?